resources/webroot/stats.mustache

```diff
@@ -221,10 +222,10 @@
             options: teamOptions
         });
 
-        pokemonChart.options.title.text = "Historical Pokemon Statistics";
-        raidChart.options.title.text = "Historical Raid Statistics";
-        questChart.options.title.text = "Historical Quest Statistics";
-        invasionChart.options.title.text = "Historical Invasion Statistics";
+        pokemonChart.options.title.text = "{{{stats_historic_pokemon}}}";
```

Comment:
this should be `{{` instead of `{{{`

Reply:
If I do that, this happens:
![image](https://user-images.githubusercontent.com/1654959/168440000-be440a65-e277-437f-9607-b734bae5a166.png)

I used three mustaches because that's the way to unescape HTML characters.

https://mustache.github.io/mustache.5.html
```
All variables are HTML escaped by default. If you want to return unescaped HTML, use the triple mustache: {{{name}}}.
```